### PR TITLE
add authentic realsense model to SITL

### PR DESCRIPTION
**Pull request by baumanta**
add a more authentic sensor to our SITL environment

### CODE REVIEW

File: local_planner/launch/local_planner_realsense.launch [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!-- SITL launch file which uses an authentic gazebo model of the Realsense camera-->
+<!-- To be able to run this complete the following steps:-->
+<!-- Clone this package into catkin_ws and build:-->
+<!-- git clone git@github.com:SyrianSpock/realsense_gazebo_plugin.git-->
+<!-- catkin build realsense_gazebo_plugin (for ROS kinetic, the kinetic-devel branch must be used)-->
+<!-- Add to Gazebo model path:-->
+<!-- export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:/catkin_ws/src/realsense_gazebo_plugin/models-->
```
Comment (jkflying) [inline local_planner/launch/local_planner_realsense.launch]: Use a dummy path for the username etc

File: sim/models/iris_realsense/iris_depth_camera.sdf [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<sdf version='1.5'>
+  <model name='iris_depth_camera'>
```
Comment (nicovanduijn) [inline sim/models/iris_realsense/iris_depth_camera.sdf]: rename to `iris_realsense`

Comment (nicovanduijn) [reply sim/models/iris_realsense/iris_depth_camera.sdf]: Also rename the file to `iris_realsense.sdf`

File: sim/models/iris_realsense/model.config [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<model>
+  <name>3DR Iris</name>
+  <version>1.0</version>
+  <sdf version='1.4'>iris_depth_camera.sdf</sdf>
```
Comment (nicovanduijn) [inline sim/models/iris_realsense/model.config]: iris_realsense

File: sim/models/iris_realsense/model.config [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  <author>
+   <name>Lorenz Meier and Thomas Gubler</name>
```
Comment (nicovanduijn) [inline sim/models/iris_realsense/model.config]: Tanja Baumann ;)

File: sim/models/iris_realsense/model.config [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+   <email>lorenz@px4.io</email>
+  </author>
+
+  <description>
+    This is a model of the 3DR Iris Quadrotor. The original model has been created by
+    Thomas Gubler and is maintained by Lorenz Meier.
```
Comment (nicovanduijn) [inline sim/models/iris_realsense/model.config]: mention that it is the regular Iris (created by Lorenz & Thomas) and you added the sensor from <name of that repo>

**Pull request merged**